docs/auth.md

```diff
@@ -1,5 +1,39 @@
-# Authentication
+# Authentication & Authorization
+
+## Authentication
 
 We are using [lucia](https://lucia-auth.com/) to manage authentication and session. Lucia associates keys with each user. The primary key is the user's email and hashed password. Secondary temporary keys can be added to single use tokens.
 
 Lucia sets a cookie `auth_session` to track the users session between requests. Session data is stored in the database.
+
+## Access Control
+
+Access control is managed by [casl](https://casl.js.org/v6/en) policies. Casl policies contains actions and subjects, and a policy is generated for a user that describes which actions on which subjects a user has access to. Each action in the system declares the action and subject needed to perform the action.
+
+### Actions
+
+We define the following actions for both the web client and the api server. The policy can narrow the range of resources the action applies to.
+
+- `create` - The user can create new resources of the subject type
+- `read` - The user can read resources of the subject type
+- `administer` - The user can access admin level actions on resources of the subject type.
+- `translate` - This applies to the `Language` subject type only. The user can translate a language.
+
+### Subjects
+
+While both the api server and the web client have `Language` and `User/AuthUser` subjects, how these subjects are narrowed works differently. On the server, a prisma query can be used to narrow the access to actions on particular subjects. On the web client, the filter is limited to lists of subject IDs generated by session data.
+
+### Policy
+
+At present the policy is this:
+
+- Platform Admins - can do anything except translate a language
+- Language Admin - can view language translations, and manage language members
```

Comment:
The admin role goes past managing users for a language, they can also update it's metadata (currently just the name), and import glosses. Maybe we should include some kind of wording the encompasses those actions? 

```suggestion
- Language Admin - can view language translations, manage language information/members, and import glosses from the previous site
```